src/handlers/deeplinks.ts

```diff
@@ -97,6 +99,9 @@ export default async function handleDeeplink(
               address: addressOrENS,
             });
           } else {
+            logger.sentry('Failed to handle url', url);
+            const error = new Error('Invalid deeplink');
+            captureException(error);
```

Comment:
```suggestion
            const error = new Error('Invalid deeplink: ' + url);
            captureException(error);
```


hb this?

Reply:
Looks better to me. But I am not an expert in our sentry and logging config. @brunobar79 pls double check.